### ISSUE TITLE
let_R_rconsole_width not working after its first call

### ISSUE THREAD

**Issue by aisneto** [jalvesaq/Nvim-R#718]
I'm trying to make my nvim windows such as when I open a new application (when a plot is called, for example), the size of both windows (script and console) is evenly distributed. The wiki has just the code for that:

```
let R_rconsole_width = winwidth(0) / 2
autocmd VimResized * let R_rconsole_width = winwidth(0) / 2
```

It is not working for me, however. I can change "let R_rconsole_width" in my init.vim and it will affect the console window initial size, but it does not adjust itself when resizing a window. 

The problem seems to be on the variable "R_rconsole_width" itself, instead of the autocmd event.  When I change the variable within nvim using command "let R_rconsole_width X" with any other number, the console window size remains unchanged.

![nvim](https://user-images.githubusercontent.com/105523584/222479811-b13e03fd-5e66-44b3-bddc-889f05f49e76.png)
In this image you can see that my script window was reduced to just one column after calling a plot. The command on bottom does not work at all for changing the window size. 

The only changes in my config related to nvim-r are those two lines.

**Comment by aisneto** [comment]
The issue seemed to be that `winwidth`  is set as fixed at default. Simple setting the option `let R_buffer_opts = " "` in my init.vim worked.

However, this option only seems to refer to the R console window. If I open the object browser above the console window, the issue seems to persist. Is there any equivalent option for the Object Browser window?

**Comment by jalvesaq** [comment]
We would need a function to adjust the height, width, and position of the editor, R console, Rhelp, and Object Browser windows when the terminal was resized. Then, this function could be called on `VimResized` event. This would be especially useful for people using a tilling window manager, such as _Sway_.

I have used _Sway_ for a couple of months but I returned to Gnome. In my case, I had the following in my `~/.config/sway/config`:

```
for_window [class="R_x11"] floating enable, border none, move position 1057 0, focus mode_toggle
```

That is, the R graphics window didn't make the terminal window resize.

I can't write the function to adjust the windows because I wouldn't use it. It will be better if someone else writes and maintains it.
